src/components/link/link.component.tsx

```diff
@@ -160,13 +161,16 @@ export const Link = React.forwardRef<
           ? {
               ...componentProps,
               ...buttonProps,
-              placeholderTabIndex,
+              onFocus: () => setHasFocus(true),
```

Comment:
**suggestion** (non-blocking): you could move these `onFocus` and `onBlur` into `componentProps` so you don't have to repeat them in both objects. This seems especially sensible to me since event handlers like `onClick` and `onKeyDown` are already there.